### DISCUSSION TITLE
Rerun failed Tests

### DISCUSSION THREAD

**Discussion by sudgarre** [reqnroll/Reqnroll#301] [Q&A]
Hi All, Have anyone implemented rerunning of failed tests using reqnroll.

**Comment by gasparnagy** [comment]
See https://github.com/chrisbillson/NUnitRetry.ReqnrollPlugin or https://github.com/JoshKeegan/xRetry

**Reply by gasparnagy** [reply]
Also https://github.com/adrichem/ReqnrollRetryPlugin
